### PR TITLE
plugin: jfrog artifactory

### PR DESCRIPTION
**Pull request by rssnyder**
artifactory plugin maintained by harness.

customer support directed a customer to use this plugin, but they were unable to find documentation for it.

Adding it here with the docs so users can see all the options without looking at the code.

There is a current `artifactory` plugin so I prefixed this with `jfrog`, let me know if you have other ideas for conflicting plugins.

### CODE REVIEW

File: plugins/jfrog-artifactory/content.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+title: JFrog Artifactory
+author: harness
+tags:
+  - publish
+  - artifactory
+logo: artifactory.svg
+repo: https://github.com/harness/drone-artifactory
+image: https://hub.docker.com/r/plugins/artifactory
+license: None
```
Comment (d1wilko) [inline plugins/jfrog-artifactory/content.yaml:9]: Can you add the Blue oak license here?

**Review by d1wilko — CHANGES_REQUESTED** [review]
1 small change and I think it is good to go :)

File: plugins/jfrog-artifactory/content.yaml (hunk 1 above)
Comment (rssnyder) [reply plugins/jfrog-artifactory/content.yaml]: https://github.com/drone/drone-plugin-index/pull/389/commits/03902f5c4e3f7966f78063dbfcaeee73db6f665f

**Review by d1wilko — APPROVED** [review]
LGTM

**Pull request merged**